app-sso/service-operators/storage.hbs.md

```diff
@@ -14,6 +14,25 @@ encrypted client-server communication -- AppSSO enforces TLS by default.
 `.spec.storage` is not set.
 </p>
 
+<p class="note">
+<strong>Note:</strong>
+While data in motion is encrypted through TLS, data at rest is _not_ encrypted by default through AppSSO. Each storage
```

Comment:
> While data in motion is encrypted through TLS, data at rest is _not_ encrypted by default through AppSSO
**[thought, non-blocking]:** Almost suggests that `AuthServer` _could_ be configured to encrypt at rest. Maybe I am reading that into it.

Reply:
Mmm.. Ok, yeah. Maybe we need to re-word.